### PR TITLE
Edits to Python PlatformContent and adoption of common GS by Django

### PR DESCRIPTION
**Pull request by PeloWriter**
This PR mildly edits the Python PlatformContent. In addition, Django now uses the common Getting Started.

Note that some details for Django now reside in Troubleshooting. If that's not the right location, we can certainly move them.

Also, with the introduction of the primer content, the start of the page seems perhaps too cluttered with Notes. Feedback welcome.

Screen Shot 2020-10-02 at 5 07 39 PM

### CODE REVIEW

File: src/platforms/python/common/troubleshooting.mdx [hunk 1]
```diff
@@ -114,3 +115,58 @@ be fixed from within the SDK.
...
 20.5](https://github.com/gevent/gevent/issues/1407) but continues to be one for
 eventlet.
+
+## Django Channels
+
+A Django application using Channels 2.0 will be correctly instrumented under Python 3.7. For older versions of Python, install `aiocontextvars` from PyPI or your application will not start.
+
+If you experience memory leaks in your channels' consumers while using the SDK, you need to wrap your entire application in [Sentry's ASGI middleware](/guides/asgi/).Unfortunately the SDK is not able to do so by itself, as [Channels is missing some hooks for instrumentation](https://github.com/django/channels/issues/1348).
+
+## Matching Behavior in the Django integration
```
Comment (untitaker) [inline src/platforms/python/common/troubleshooting.mdx:125]: everything starting frmo here to the bottom of the page should not be visible outside of Django and is IMO misplaced in Troubleshooting

File: src/includes/getting-started-config/python.django.mdx [hunk 2]
```diff
@@ -1,4 +1,13 @@
-To configure the SDK, initialize it with the Django integration in your `settings.py` file:
+Initialize the Python SDK with the Django integration in your `settings.py` file. This snippet:
+- Instruments your Django integration to [monitor the performance](/platforms/javascript/performance/) of your application.
```
Comment (untitaker) [inline src/includes/getting-started-config/python.django.mdx:2]: It also does error monitoring. Is it necessary to mention anything at all this prominently?

File: src/platforms/python/common/troubleshooting.mdx [hunk 3]
```diff
@@ -114,3 +115,58 @@ be fixed from within the SDK.
 This [issue has been fixed with gevent
 20.5](https://github.com/gevent/gevent/issues/1407) but continues to be one for
 eventlet.
+
+## Django Channels
```
Comment (untitaker) [inline src/platforms/python/common/troubleshooting.mdx:119]: This should not be in common Python. Perhaps we can abuse PlatformContent and create a new type of platform content called python-troubleshooting-extra?

@dcramer wdyt

**Review by untitaker — CHANGES_REQUESTED** [review]
minor changes only

File: src/platforms/python/common/troubleshooting.mdx [hunk 4]
```diff
@@ -114,3 +115,58 @@ be fixed from within the SDK.
...
+
+A Django application using Channels 2.0 will be correctly instrumented under Python 3.7. For older versions of Python, install `aiocontextvars` from PyPI or your application will not start.
+
+If you experience memory leaks in your channels' consumers while using the SDK, you need to wrap your entire application in [Sentry's ASGI middleware](/guides/asgi/).Unfortunately the SDK is not able to do so by itself, as [Channels is missing some hooks for instrumentation](https://github.com/django/channels/issues/1348).
```
Comment (untitaker) [inline src/platforms/python/common/troubleshooting.mdx:123]:
```suggestion
If you experience memory leaks in your channels' consumers while using the SDK, you need to wrap your entire application in [Sentry's ASGI middleware](/platforms/python/guides/asgi/).Unfortunately the SDK is not able to do so by itself, as [Channels is missing some hooks for instrumentation](https://github.com/django/channels/issues/1348).
```

File: src/platforms/python/common/troubleshooting.mdx (hunk 1 above)
Comment (PeloWriter) [reply src/platforms/python/common/troubleshooting.mdx:125]: I'll move some of this into the verification step, and try the PlatformSection toggle for what remains.

File: src/includes/getting-started-config/python.django.mdx (hunk 2 above)
Comment (PeloWriter) [reply src/includes/getting-started-config/python.django.mdx:2]: We say that immediately prior in this sentence "Once this is done, the Sentry SDK captures all unhandled exceptions." 

We keep this in the `include` as performance monitoring under development for other SDKs.

File: src/platforms/python/common/troubleshooting.mdx (hunk 3 above)
Comment (PeloWriter) [reply src/platforms/python/common/troubleshooting.mdx:119]: I will try a PlatformSection here.

**Pull request merged**